hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/replication/health/TestClosingContainerHandler.java

```diff
@@ -153,32 +164,59 @@ public void testUnhealthyRatisReplicaIsNotClosed() {
   /**
    * Close commands should be sent for Open or Closing replicas.
    */
-  @Test
-  public void testOpenOrClosingReplicasAreClosed() {
+  @ParameterizedTest
+  @MethodSource("replicationConfigs")
+  public void testOpenOrClosingReplicasAreClosed(ReplicationConfig repConfig) {
```

Comment:
Parameterizing this is a good idea. If I understand correctly, this test will now run for both EC and RATIS replication configs. If so, we can delete the next test `testOpenOrClosingRatisReplicasAreClosed()`.

Reply:
Yeah, it's covered by the parameterized test.
Thanks for pointing this out.